components/camera/camera_test.go

```diff
@@ -184,25 +184,29 @@ func TestReconfigurableCamera(t *testing.T) {
 	actualCamera2 := &mock{Name: testCameraName2}
 	reconfCamera2, err := camera.WrapWithReconfigurable(actualCamera2, resource.Name{})
 	test.That(t, err, test.ShouldBeNil)
-	test.That(t, actualCamera1.reconfCount, test.ShouldEqual, 0)
```

Comment:
This tests needed to be updated since we no longer expect `Reconfigure` to close `actualCamera1` or any cameras. I substituted the stale test with new ones as needed.